experiments/test.py

```diff
@@ -0,0 +1,65 @@
+import requests
+
+
+agent_endpoints = {
+    "vietnam": "http://localhost:8001",
+    "thailand": "http://localhost:8002",
+    "main": "http://localhost:8000",
+}
+
+if __name__ == "__main__":
+    question = "Give me some places in South East Asia for vacation"
```

Comment:
ask user for country in one line and generate prompt

Reply:
> ask user for country in one line and generate prompt

you mean generating a message/request for other agent right? Because we already generated the prompt in .env file when initalizing the container.